packages/salesforcedx-apex-debugger/src/core/breakpointService.ts

```diff
@@ -219,7 +228,62 @@ export class BreakpointService {
     return Promise.resolve(this.getBreakpointsFor(uri));
   }
 
+  public async createExceptionBreakpoint(
+    projectPath: string,
+    sessionId: string,
+    typeref: string
+  ): Promise<string | undefined> {
+    const execution = new CliCommandExecutor(
+      new SfdxCommandBuilder()
+        .withArg('force:data:record:create')
+        .withFlag('--sobjecttype', 'ApexDebuggerBreakpoint')
+        .withFlag(
+          '--values',
+          `SessionId='${sessionId}' FileName='${typeref}' IsEnabled='true' Type='Exception'`
+        )
+        .withArg('--usetoolingapi')
+        .withArg('--json')
+        .build(),
+      { cwd: projectPath, env: RequestService.getEnvVars() }
+    ).execute();
```

Comment:
CLI command to create an ApexDebuggerBreakpoint object with for an exception type.